client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -54,6 +55,21 @@ export default function StepStoreAddress( props: WooCommerceInstallProps ): Reac
 		update( WOOCOMMERCE_DEFAULT_COUNTRY, event.target.value );
 	};
 
+	// @todo: Add a general hook to get and update multi-option data like the profile.
```

Comment:
I had a look at this yesterday, was thinking instead of exposing `WOOCOMMERCE_ONBOARDING_PROFILE`, exposing `WOOCOMMERCE_STORE_EMAIL` at the hook level. Then inside the hook, conditionally managing access to the profile data. This should lead to some simple code in the hook but I think I ran into issues with the get/update functions leveraging the const/types so that needs breaking first.

Not suggesting we change this now, just dumping some thoughts in case it helps when/if we change this later.